.gitignore

```diff
@@ -1,2 +1,8 @@
 fuel/app/logs/*/*/*
 fuel/app/cache/*/*
+coverage-html/
+.env
+*.tmp
+fuel/vendor/
+fuel/app/config/*/migrations.php
```

Comment:
@luong-phuc-mulodo ah.. sorry. i misunderstood the file .DS_Store added to a directory.